src/actions/portfolio/handlers/eth/index.js

```diff
@@ -18,23 +18,25 @@ export const getEthPortfolio = async (address, fiatCurrency, coinGeckoIds) => {
     if (tokens && tokens.length > 0) {
         for (const token of json.tokens) {
             const { symbol, decimals } = token.tokenInfo;
-            const coinGeckoId = coinGeckoIds[symbol.toLowerCase()];
-            if (!coinGeckoId) {
-                console.warn(`could not get coingecko id for symbol ${symbol}`);
-                continue;
+            if (symbol) {
```

Comment:
To reduce nesting levels and improve code readability, I feel like it would be better to do something like:
```
if (!symbol) {
    continue;
}
```
Plus, how do you feel about adding the same kind of check in all the other handlers too? (if applicable)

Reply:
I feel like too much effort sir

Reply:
If the same check has to be done in different files probably the code should be refactor a bit, but unfortunately I don't have time right now. Good luck and keep up the good work sir